src/components/App.js

```diff
@@ -1,11 +1,34 @@
+import React, { Component } from 'react';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
+import calculate from '../logic/calculate';
 
-const App = () => (
-  <>
-    <Display />
-    <ButtonPanel />
-  </>
-);
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: null,
+      next: null,
+      operation: null,
+    };
+
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(buttonName) {
```

Comment:
- [ ] It would be better if the button respond to the click event. Now, none of the buttons update the state of the application and it also returns an error => "Uncaught Error: setState(...): takes an object of state variables to update or a function which returns an object of state variables."